packages/@aws-cdk/aws-ec2/lib/security-group.ts

```diff
@@ -404,6 +404,7 @@ export class SecurityGroup extends SecurityGroupBase {
       // In the case of "allowAllOutbound", we don't add any more rules. There
       // is only one rule which allows all traffic and that subsumes any other
       // rule.
+      this.node.addWarning('Ignoring Egress rule since \'allowAllOutbound\' is set to true; To add customize rules, set allowAllOutbound=false on the SecurityGroup');
```

Comment:
Is it possible to include some info about which egress rule is ignored?

Reply:
we can add, the `CIDR` but Im not sure this will make a different since all rules are ignored when `allowAllOutbound` is set to true.